docker/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
We don't need this - already have one in the root folder.

Reply:
Add a license herder to the dockerfile and the script files instead

Reply:
ok.